app/views/article/_index_default.html.erb

```diff
@@ -22,7 +22,14 @@
 <% if document.access_panels.online? && !document.eds_restricted? %>
   <ul class='document-metadata dl-horizontal dl-invert'>
     <% document.access_panels.online.links.each do |link| %>
-      <li><%= link_to(link.text, link.href) %></li>
+      <li>
+        <% if link.text =~ /^Find it in print/ -%>
```

Comment:
I wonder if we should move this up to the `Link` class somehow given that it's re-used in several places.

I may have confused things by overloading the `sfx` accessor for the class as a broad term for the particular SFX links that we use to pull in data into the async. loaded access panel I created.

If we did go this route, we may need to add an accessor unless one of the existing ones seems like a good fit, and either use that for this purpose.

Thoughts?